lib/tasks/bundlers/generateFlexChangesBundle.js

```diff
@@ -44,27 +46,26 @@ module.exports = function({workspace, options}) {
 	}
 
 	async function updateFLdependency() {
-		const manifestResource = await workspace.byPath(`${pathPrefix}/manifest.json`);
```

Comment:
I guess my review comments created more confusion than help. My point was that the same task (read manifest resource as buffer, wait for it, parse as JSON) are implemented in two consecutive method in two different ways. Using the much more readable lines 47 and 48 also instead of lines 57-68 was what I had in mind.

Reply:
Thanks. Hope it is ok now